x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -331,7 +331,7 @@ export const AddExceptionFlyoutWrapper: React.FC<AddExceptionFlyoutWrapperProps>
   /**
    * This should be re-visited after UEBA work is merged
```

Comment:
@dhurley14 just wanted to confirm these changes don't affect anything from UEBA per your comment here?

Reply:
That comment's been there for a while I think - wasn't @dhurley14 that wrote it. Unfortunately, he's out till next week. Do you know who could look at this from UEBA?

Reply:
@stephmilovic do you know who would have context on this?